blocks/columns/columns.css

```diff
@@ -161,6 +161,13 @@
   max-width: 1200px;
 }
 
+.columns.heading-divider h2,
+.columns.heading-divider h3,
+.columns.heading-divider h4 {
+  border-bottom: 1px solid #3c4650;
```

Comment:
The color is available as a variable here: https://github.com/hlxsites/sunstar/blob/main/styles/styles.css#L59